src/toBeReviewed.js

```diff
@@ -0,0 +1,26 @@
+var animalData = [
+  {
+    id: 1,
+    title: "hippo",
+    faveFood: "carrots",
+  },
+  {
+    id: 2,
+    title: "Cat",
+    faveFood: "carrots",
+  },
+  {
+    id: 3,
+    title: "ducks",
+    faveFood: "breadcrumbs",
+  },
+];
+
+exports.findAnimal = function () {
+  for (x = 0; x < 3; x++) {
+    if ((animalData[x].title = args[0])) {
```

Comment:
Firstly, I believe "args" is referring to the arguments object, which should be "arguments".  Secondly, unless special  reason, the better practice suggests using the explicit argument, like "title" for functions as mentioned above.